platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/service/WritablePlatformStateStore.java

```diff
@@ -75,7 +75,7 @@ public WritablePlatformStateStore(@NonNull final WritableStates writableStates)
     /**
      * Overwrite the current platform state with the provided state.
      */
-    public void setAllFrom(@NonNull final PlatformStateAccessor accessor) {
+    public void setAllFrom(@NonNull final PlatformStateModifier accessor) {
```

Comment:
nit: param still called `accessor`, but this is now a `modifier`

Reply:
Fixed